webpack.config.js

```diff
@@ -1,7 +1,9 @@
 const path = require('path')
+const env = require('./server/config').env
+const inDev = env === 'dev' || env === 'development'
 module.exports = (env, argv) => ({
-  mode: 'development', // 'development' or 'production',
-  devtool: 'source-map', // 'source-map' or 'none',
+  mode: !inDev ? 'production' : 'development',
```

Comment:
Is there a reason the negation is needed? Would `inDev ? 'development' : 'production'` be sufficient.
Same comment applies below.

Reply:
I think the reason I made it negative is the majority of environments will be truthy, leaving only development to be falsey.  Happy to switch if readability is a concern

Reply:
If `inDev` came from, say, an env var where it could be `null`, `undefined`, `false`, etc then maybe but given the definition (`const inDev = env === 'dev' || env === 'development'`) it is only going to be `true` or `false`.

I'll approved it and leave it for you to decide if you want to change it.